.golangci.yml

```diff
@@ -33,6 +32,12 @@ linters:
 
 issues:
   exclude-rules:
+    - text: "is unused for linter"
```

Comment:
This is to fix [these errors](https://github.com/cosmos/ibc-go/actions/runs/4590155929/jobs/8105523498?pr=3354).